<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test</class>
 <widget class="QWidget" name="test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/paper-background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>681</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
    border-radius: 15px; 
    background-color: transparent
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/vintage-wallpaper-background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(123, 156, 100);
    color: white; /* Цвет текста */
    border: none; /* Без рамки */
    border-radius: 10px; /* Закругленные границы */
    padding: 10px 20px; /* Отступы внутри кнопки */
    font-size: 16px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: rgb(255, 174, 205); 
    cursor: pointer; /* Изменение курсора при наведении */
}</string>
   </property>
   <property name="text">
    <string>нет, это не совсем так</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(123, 156, 100);
    color: white; /* Цвет текста */
    border: none; /* Без рамки */
    border-radius: 10px; /* Закругленные границы */
    padding: 10px 20px; /* Отступы внутри кнопки */
    font-size: 16px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: rgb(255, 174, 205); 
    cursor: pointer; /* Изменение курсора при наведении */
}</string>
   </property>
   <property name="text">
    <string>пожалуй, так</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(123, 156, 100);
    color: white; /* Цвет текста */
    border: none; /* Без рамки */
    border-radius: 10px; /* Закругленные границы */
    padding: 10px 20px; /* Отступы внутри кнопки */
    font-size: 16px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: rgb(255, 174, 205); 
    cursor: pointer; /* Изменение курсора при наведении */
}</string>
   </property>
   <property name="text">
    <string>верно</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color:rgb(123, 156, 100);
    color: white; /* Цвет текста */
    border: none; /* Без рамки */
    border-radius: 10px; /* Закругленные границы */
    padding: 10px 20px; /* Отступы внутри кнопки */
    font-size: 16px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: rgb(255, 174, 205); 
    cursor: pointer; /* Изменение курсора при наведении */
}</string>
   </property>
   <property name="text">
    <string>совершенно верно</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
    border-radius: 15px; 
    background-color: transparent
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/sun (1).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>200</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
    border-radius: 15px; 
    background-color: transparent
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/rainy.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="introText_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>17</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: none;
    background: none;
    color: rgb(123, 156, 100);
}</string>
   </property>
   <property name="text">
    <string>Ответьте на вопрос, выбрав один из предложенных вариантов ответа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: none;
    background: none;
    color: rgb(123, 156, 100);
}
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
